Translator/Translator/Header/Triangle.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "Point.h"
+
+class Triangle {
+public:
+    Point p1;
+    Point p2;
+    Point p3;
+
+    Triangle(Point p1, Point p2, Point p3);
+    void getTriangle();

```

Comment:
what do you mean by void, take it as int so that it can return the values to the next class